.github/workflows/ci.yml

```
@@ -59,6 +59,12 @@ jobs:
     - name: Run Playwright tests
       run: yarn test:e2e
     
+    - name: Upload visual tests to Percy
+      run: yarn percy upload visual-tests/
+      env:
+        PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+      if: always() && hashFiles('visual-tests/*.png') != ''
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)